mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java

```diff
@@ -195,14 +198,35 @@ public void writeReceiptToPDF(ResponseBody responseBody, String filename) {
         if (!mifosDirectory.exists()) {
             mifosDirectory.mkdirs();
         }
-        File documentFile = new File(mifosDirectory.getPath(), filename);
+        final File documentFile = new File(mifosDirectory.getPath(), filename);
         if (!FileUtils.writeInputStreamDataToFile(responseBody.byteStream(), documentFile)) {
             showToast(Constants.ERROR_DOWNLOADING_RECEIPT);
         } else {
-            showSnackbar(Constants.RECEIPT_DOWNLOADED_SUCCESSFULLY);
+            Snackbar.make(findViewById(android.R.id.content),
```

Comment:
Please the utils lib `Toaster` that we have in the project.

Reply:
@luckyman20 I initially tried using the Toaster class but it did not have the method for creating snackbar with an action so I had to use the Snackbar class directly.

Reply:
Add that in the `Toaster` class and then use it. Don't forget to mention it in commit description.